apps/web/src/api/hooks/notification-templates/useFetchBlueprints.ts

```diff
@@ -0,0 +1,148 @@
+      {
+        id: '4',
+        name: ':fa-regular fa-hand: Welcome message',
+        description:
+          'Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quas totam quod beatae. Ipsam quasi fugiat commodi adipisci eligendi necessitatibus cumque aliquam, dicta natus cupiditate suscipit voluptatum rerum debitis. Ipsum!',
+      },
+      {
+        id: '5',
+        name: ':fa-solid fa-envelope-open-text: Invite message',
+        description:
+          'Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quas totam quod beatae. Ipsam quasi fugiat commodi adipisci eligendi necessitatibus cumque aliquam, dicta natus cupiditate suscipit voluptatum rerum debitis. Ipsum!',
+      },
+      {
+        id: '6',
+        name: ':fa-solid fa-gift: Refferal link',
+        description:
+          'Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quas totam quod beatae. Ipsam quasi fugiat commodi adipisci eligendi necessitatibus cumque aliquam, dicta natus cupiditate suscipit voluptatum rerum debitis. Ipsum!',
+      },
+    ],
+  },
+  {
+    name: 'Authentification',
+    templates: [
+      {
+        id: '7',
+        name: ':fa-solid fa-wand-magic-sparkles: Magic link',
+        description:
+          'Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quas totam quod beatae. Ipsam quasi fugiat commodi adipisci eligendi necessitatibus cumque aliquam, dicta natus cupiditate suscipit voluptatum rerum debitis. Ipsum!',
+      },
+      {
+        id: '8',
+        name: ':fa-solid fa-unlock: Password change',
+        description:
+          'Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quas totam quod beatae. Ipsam quasi fugiat commodi adipisci eligendi necessitatibus cumque aliquam, dicta natus cupiditate suscipit voluptatum rerum debitis. Ipsum!',
+      },
+      {
+        id: '9',
+        name: ':fa-solid fa-unlock: Password change2',
+        description:
+          'Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quas totam quod beatae. Ipsam quasi fugiat commodi adipisci eligendi necessitatibus cumque aliquam, dicta natus cupiditate suscipit voluptatum rerum debitis. Ipsum!',
+      },
+      {
+        id: '10',
+        name: ':fa-solid fa-unlock: Password change3',
+        description:
+          'Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quas totam quod beatae. Ipsam quasi fugiat commodi adipisci eligendi necessitatibus cumque aliquam, dicta natus cupiditate suscipit voluptatum rerum debitis. Ipsum!',
+      },
+    ],
+  },
+];
+
+const getTemplateDetails = (templateName: string): { name: string; iconName: IconName } => {
+  const regexResult = /^:.{1,}: /.exec(templateName);
+  let name = '';
+  let iconName = 'fa-solid fa-question';
+  if (regexResult !== null) {
+    name = templateName.replace(regexResult[0], '').trim();
+    iconName = regexResult[0].replace(/:/g, '').trim();
```

Comment:
Not for now but here we could have problems if in the future anyone adds 2 icon placeholder as:
```
/^:.{1,}: /.exec(':fa-solid fa-unlock: :fa-solid fa-zap: template name'));
```
would return `:fa-solid fa-unlock: :fa-solid fa-zap: ` and the global replace for the iconName would become `fa-solid fa-unlock fa-solid fa-zap` that it won't be processable I think.

Reply:
We only show one icon. The blueprints will be created by us and updated really rarely, we've agreed with Dima on this particular icon notation in the template name.
But in any case the last one `fa-solid fa-zap` will override the previous one `fa-solid fa-unlock`.